src/stores/messages/MessagesDataProviderStore.ts

```diff
@@ -100,7 +100,8 @@ export default class MessagesDataProviderStore {
 		return (
 			this.isLoadingNextMessages ||
 			this.isLoadingPreviousMessages ||
-			Boolean(this.anchorChannel?.isLoading)
+			Boolean(this.anchorChannel?.isLoading) ||
+			Boolean(this.messageAC)
```

Comment:
messageAC isn't observable
so this computed property will not react when it's changed
maybe we should add `isLoadingAnchorMessage` state?